packages/runtime/src/lib/txn.ts

```diff
@@ -221,6 +221,14 @@ export function txAppArg(
 ): StackElem {
 	const tealVersion: number = interpreter.tealVersion;
 
+	if (txField === "Logs") {
```

Comment:
I think we should move it to [txnSpecByField](https://github.com/scale-it/algo-builder/blob/29657c46caaefa4b8869677e33a69623ca55b11c/packages/runtime/src/lib/txn.ts#L99). get i
Because we don't get it from specific field from Txn object.

Reply:
yes

Reply:
I think we need it in both cases. We can do `gitxna 0 Logs 1` and `gitxnas 0 Logs`

Reply:
After discuss, we will keep this line at here. However we will refactor it in future